src/ImgixTag.js

```diff
@@ -7,45 +7,55 @@ var ImgixTag = (function() {
     this.settings = opts || {};
 
     if (!this.el) {
-      throw new Error('ImgixTag must be passed a DOM element.');
-    }
-
-    if (this.el.hasAttribute('ix-initialized') && !this.settings.force) {
-      return;
-    }
-
-    this.ixPathVal = el.getAttribute(this.settings.pathInputAttribute);
-    this.ixParamsVal = el.getAttribute(this.settings.paramsInputAttribute);
-    this.ixSrcVal = el.getAttribute(this.settings.srcInputAttribute);
-    this.ixHostVal =
-      el.getAttribute(this.settings.hostInputAttribute) || this.settings.host;
-
-    if (this.ixPathVal && !this.ixHostVal) {
-      throw new Error(
-        'You must set a value for `imgix.config.host` or specify an `ix-host` attribute to use `ix-path` and `ix-params`.'
-      );
-    }
+      console.warn('ImgixTag must be passed a DOM element.');
+    } else {
```

Comment:
Rather than wrapping essentially ~all of this function in an `else` block, can we just do an early return in the initial case?

```js
if (!this.el) {
  console.warn('ImgixTag must be passed a DOM element.');
  return;
}

// continue rest of constructor
```

Reply:
Agreed 👍